docs/sources/tempo/traceql/metrics-queries/functions.md

```diff
@@ -97,16 +77,29 @@ down by HTTP route.
 This might let you determine that `/api/sad` had a higher rate of erroring
 spans than `/api/happy`, for example.
 
-### The `count_over_time`, `min_over_time` and `max_over_time` functions
+## The `count_over_time` function
 
 The `count_over_time()` let you counts the number of matching spans per time interval.
+For more information, refer to the [`step` API parameter](https://grafana.com/docs/tempo/<TEMPO_VERSION>/api_docs/#traceql-metrics).
+
+
+### Example
 
 ```
 { name = "GET /:endpoint" } | count_over_time() by (span.http.status_code)
 
 ```
 
+## The `min_over_time` function
+
 The `min_over_time()` let you aggregate numerical values by computing the minimum value of them, such as the all important span duration.
+For more information, refer to the [`step` API parameter](https://grafana.com/docs/tempo/<TEMPO_VERSION>/api_docs/#traceql-metrics).
+
+### Parameters
+
+None.
```

Comment:
Are there any parameters fro this?

Reply:
i believe this takes a field name it's min-ing?

your examples below show this

Reply:
Done.